docs/tutorials/mongo_maintenance.rst

```diff
@@ -0,0 +1,82 @@
+Mongo Maintenance
+=================
+
```

Comment:
Start out explaining who this is for and when they may need to use it.  i.e., 

1.  if you are not using a mongo back-end, you can skip.   
2. Then give a roadmap from a high level.  What are the objectives that the person is trying to accomplish, so something like 
    1. set up a mongo instance, 
        1. use mongo atlas 
        1. run your own mongo locally
    2. move existing yml or json fs databases over
    3.  link regolith to the mongo db

and so on.  i.e, a brief roadmap of what the high-level objectives are, and then dive in to the step by step things to get you there.

Reply:
Addressed